internal/pkg/controllers/spod/bindata/spod.go

```diff
@@ -388,6 +436,15 @@ var Manifest = &appsv1.DaemonSet{
 							},
 						},
 					},
+					{
+						Name: "host-devkmsg-volume",
+						VolumeSource: v1.VolumeSource{
+							HostPath: &v1.HostPathVolumeSource{
+								Path: "/dev/kmsg",
```

Comment:
```suggestion
								Path: KmsgDropDirectory,
```

Reply:
fixed it and renamed the variable to make it more generic so we can use as both volume path and drop path.